raco/expression/aggregate.py

```diff
@@ -203,6 +203,15 @@ def typeof(self, scheme, state_scheme):
         return self.input.typeof(scheme, state_scheme)
 
 
+class MULTIMIN(NaryFunction, TrivialAggregateExpression):
```

Comment:
I'm happy to see `MIN` taking multiple columns but sometimes the column names are unknown (see above discussion).